nixos/modules/services/web-servers/minio.nix

```diff
@@ -29,6 +29,38 @@ in
       description = "The config directory, for the access keys and other settings.";
     };
 
+    accessKey = mkOption {
+      default = "";
+      type = types.str;
+      description = ''
+        Specify the Secret key of 8 to 40 characters in length that clients use to access the server.
+        If not set the generated key from <literal>configDir</literal> is used.
```

Comment:
There should be an option to not store secrets in the nix store. For background: https://github.com/NixOS/nixpkgs/issues/24288